apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorBaseInfoSection.vue

```diff
@@ -138,14 +138,18 @@ const state = reactive({
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const isLatestVersion = (version: string) => true;
 
-const fetchCollectorUpdate = async (): Promise<CollectorModel> => SpaceConnector.client.inventory.collector.update({
-    collector_id: collectorFormStore.collectorId,
-    plugin_info: {
-        version: collectorFormState.version,
-        upgrade_mode: collectorFormState.autoUpgrade ? 'AUTO' : 'MANUAL',
-    },
-    tags: collectorFormState.tags,
-});
+const fetchCollectorUpdate = async (): Promise<CollectorModel> => {
+    if (!collectorFormStore.collectorId) throw new Error('collector_id is required');
```

Comment:
No, we don't have to. :)
`ErrorHandler.handleRequestError()` method decides whether to pop up the toast with translated message we given. 
It toasts only when the error is the bad-request error(status 400) or the server error(status 500).
In case of the server error, it shows fixed message like 'Something is Wrong! Contact to administrator.'.
**But in the bad-request error case, it shows the message we given.**

The case I made is none of the case for showing toast. This is only for developers to check edge cases or bugs.